src/lib.rs

```diff
@@ -1,10 +1,48 @@
+//! Epoch-based memory reclamation.
+//!
+//! # Pointers
+//!
+//! Concurrent collections are built using atomic pointers. This module provides [`Atomic`], which
+//! is just a shared atomic pointer to a heap-allocated object. Loading an [`Atomic`] yields a
+//! [`Ptr`], which is an epoch-protected pointer through which the loaded object can be safely read.
+//!
+//! # Pinning
+//!
+//! Before an [`Atomic`] can be loaded, the current mutator must be [`pin`]ned. By pinning a mutator
```

Comment:
@Vtec234 yes, it is a little bit unfortunate..  In the future, each `Realm` should be able to make a new `Mutator`, and in that case we no longer need to access the global `REGISTRIES` and `EPOCH` from `mutator.rs`, as done in #3.  Note that `mutator.rs` there does not reference the global data.